evalai/utils/challenges.py

```diff
@@ -0,0 +1,64 @@
+import json
+import requests
+
+from click import echo
+
+from pylsy import pylsytable
+
+from evalai.utils.auth import get_headers
+from evalai.utils.urls import urls
+from evalai.utils.common import valid_token
+
+
+DOMAIN = "http://localhost:8000"
```

Comment:
please change the variable name to API_HOST_URL. Also, modify it to:
`API_HOST_URL = os.environ.get("EVALAI_API_URL", 'http://localhost:8000')`